From 47e01adce3fad3cea2744d1fa473922831d05ddc Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@gmail.com>
Date: Wed, 11 May 2022 16:05:22 +0200
Subject: [PATCH 5/6] Avoid truncation assignment warnings of LOG2 and PI

---
 OgreMain/include/OgreMath.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/OgreMain/include/OgreMath.h b/OgreMain/include/OgreMath.h
index 288f0ae3d..9e8b8b56e 100644
--- a/OgreMain/include/OgreMath.h
+++ b/OgreMain/include/OgreMath.h
@@ -358,7 +358,7 @@ namespace Ogre
         static inline Real Log (Real fValue) { return std::log(fValue); }
 
         /// Stored value of log(2) for frequent use
-        static constexpr Real LOG2 = 0.69314718055994530942;
+        static constexpr Real LOG2 = Real(0.69314718055994530942);
 
         static inline Real Log2 (Real fValue) { return std::log2(fValue); }
 
@@ -740,7 +740,7 @@ namespace Ogre
 
         static constexpr Real POS_INFINITY = std::numeric_limits<Real>::infinity();
         static constexpr Real NEG_INFINITY = -std::numeric_limits<Real>::infinity();
-        static constexpr Real PI = 3.14159265358979323846;
+        static constexpr Real PI = Real(3.14159265358979323846);
         static constexpr Real TWO_PI = Real( 2.0 * PI );
         static constexpr Real HALF_PI = Real( 0.5 * PI );
         static constexpr float fDeg2Rad = PI / Real(180.0);
-- 
2.25.1

