From 237752c6192d2ae23149f84c5bf4cb260b44ed86 Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@sintef.no>
Date: Tue, 7 Jun 2022 17:20:24 +0200
Subject: [PATCH 5/5] Set rpath with origin variants

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3a51f643c..f4e0341e0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -443,8 +443,8 @@ mark_as_advanced(
 ###################################################################
 include(ConfigureBuild)
 
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_RPATH};${CMAKE_INSTALL_PREFIX}/${OGRE_LIB_DIRECTORY}")
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_RPATH};$ORIGIN;$ORIGIN/..;$ORIGIN/../lib;$ORIGIN/../lib/OGRE;$ORIGIN/../..")
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE)
 
 ###################################################################
 # disable way too common compiler warnings on project level
-- 
2.30.2

