From 5151597782a589f8e040b82822a5d5081d56ed49 Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@sintef.no>
Date: Mon, 31 Oct 2022 13:37:31 +0100
Subject: [PATCH 5/5] Set rpath with origin variants

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 14337a929..583ee547d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -453,8 +453,8 @@ mark_as_advanced(
 ###################################################################
 include(ConfigureBuild)
 
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_RPATH};${CMAKE_INSTALL_PREFIX}/${OGRE_LIB_DIRECTORY}")
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_RPATH};$ORIGIN;$ORIGIN/..;$ORIGIN/../lib;$ORIGIN/../lib/OGRE;$ORIGIN/../..")
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE)
 
 ###################################################################
 # disable way too common compiler warnings on project level
-- 
2.30.2

