From 01b243cbf65656abe1a051200adbb2bca55b3490 Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@sintef.no>
Date: Thu, 26 Oct 2023 10:44:41 +0200
Subject: [PATCH] Do not use found freetype if not requested

---
 Components/Overlay/CMakeLists.txt | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/Components/Overlay/CMakeLists.txt b/Components/Overlay/CMakeLists.txt
index 2a05228f1..95d9d8490 100644
--- a/Components/Overlay/CMakeLists.txt
+++ b/Components/Overlay/CMakeLists.txt
@@ -34,7 +34,7 @@ if(OGRE_BUILD_COMPONENT_OVERLAY_IMGUI)
     ${IMGUI_DIR}/imgui_tables.cpp
     ${IMGUI_DIR}/imgui_widgets.cpp
     ${IMGUI_DIR}/imgui_demo.cpp)
-  if(FREETYPE_FOUND)
+  if(OGRE_WITH_FREETYPE)
     set(IMGUI_ENABLE_FREETYPE TRUE)
     list(APPEND SOURCE_FILES ${IMGUI_DIR}/misc/freetype/imgui_freetype.cpp)
   endif()
@@ -50,10 +50,9 @@ target_include_directories(OgreOverlay PUBLIC
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
   $<INSTALL_INTERFACE:include/OGRE/Overlay>)
 
-if(FREETYPE_FOUND)
+if(OGRE_WITH_FREETYPE)
   target_compile_definitions(OgreOverlay PRIVATE HAVE_FREETYPE)
-  target_link_libraries(OgreOverlay PRIVATE Freetype::Freetype)#${FREETYPE_LIBRARIES})
-  #target_include_directories(OgreOverlay PRIVATE ${FREETYPE_INCLUDE_DIRS})
+  target_link_libraries(OgreOverlay PRIVATE Freetype::Freetype)
   if(ZLIB_FOUND)
     target_link_libraries(OgreOverlay PRIVATE ZLIB::ZLIB)
   endif()
-- 
2.30.2

