From 5ad82058fa9d7ea6b3c06e9a3716e1cfd0e37e4c Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@sintef.no>
Date: Fri, 14 Apr 2023 22:40:33 +0200
Subject: [PATCH 2/2] Add CTest Ogre

---
 Tests/CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Tests/CMakeLists.txt b/Tests/CMakeLists.txt
index 11da11077..8cf578510 100644
--- a/Tests/CMakeLists.txt
+++ b/Tests/CMakeLists.txt
@@ -81,7 +81,11 @@ if (OGRE_BUILD_TESTS)
     add_executable(Test_Ogre ${HEADER_FILES} ${SOURCE_FILES} ${RESOURCE_FILES} )
     ogre_install_target(Test_Ogre "" FALSE)
     target_link_libraries(Test_Ogre OgreBites Codec_STBI ${OGRE_LIBRARIES} GTest::gtest)
-    
+
+    add_test(NAME Test_Ogre
+        COMMAND $<TARGET_FILE_NAME:Test_Ogre>
+        WORKING_DIRECTORY $<TARGET_FILE_DIR:Test_Ogre>)
+
     if(ANDROID)
         set_target_properties(Test_Ogre PROPERTIES LINK_FLAGS -pie)
     endif()
-- 
2.30.2

