From a70adeee1d5082862ad3a706f1514775494c52b7 Mon Sep 17 00:00:00 2001
From: Joakim Haugen <joakim.haugen@sintef.no>
Date: Wed, 26 Jun 2024 11:19:52 +0200
Subject: [PATCH] Include zip during build only

---
 OgreMain/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OgreMain/CMakeLists.txt b/OgreMain/CMakeLists.txt
index df19a5115..483f9f3dd 100644
--- a/OgreMain/CMakeLists.txt
+++ b/OgreMain/CMakeLists.txt
@@ -130,7 +130,6 @@ if (OGRE_CONFIG_ENABLE_ASTC)
 endif ()
 
 if (OGRE_CONFIG_ENABLE_ZIP)
-  include_directories(src/zip)
   list(APPEND HEADER_FILES include/OgreZip.h)
   list(APPEND SOURCE_FILES src/OgreZip.cpp src/zip/zip.c)
 
@@ -172,6 +171,7 @@ generate_export_header(OgreMain
 target_include_directories(OgreMain PUBLIC 
   "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>" 
   "$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>"
+  "$<IF:$<BOOL:OGRE_CONFIG_ENABLE_ZIP>,$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/zip>,>"
   $<INSTALL_INTERFACE:include/OGRE>
   PRIVATE "${OGRE_PROFILING_REMOTERY_PATH}")
 
-- 
2.39.2

